app/controllers/opportunity_matches_controller.rb

```diff
@@ -75,6 +75,38 @@ def create_match_attributes(client_id)
     }
   end
 
+  def prioritized_column_data
+    @prioritized_column_data ||= Client.prioritized_columns_data
+  end
+
+  def prioritized_column_labels
+    [].tap do |result|
+      @opportunity.match_route.prioritized_client_columns.map(&:to_sym).each do |column|
+        column_data = prioritized_column_data[column]
+        next if column_data.blank?
+        next if column_data[:display_check].present? && send(column_data[:display_check]) == false
+
+        title = column_data[:title]
+        title = column_data[:alt_title] if column_data[:alt_title].present?
+        result << title
+      end
+    end
+  end
+  helper_method :prioritized_column_labels
+
+  def prioritized_column_values(client)
+    [].tap do |result|
+      @opportunity.match_route.prioritized_client_columns.map(&:to_sym).each do |column|
+        column_data = prioritized_column_data[column]
+        next if column_data.blank?
+        next if column_data[:display_check].present? && send(column_data[:display_check]) == false
```

Comment:
What is this doing?

Reply:
Oh, I see, it's calling things like `can_view_dob?`  I think this might be cleaner/clearer as:
```suggestion
        next if column_data[:display_check].present? && ! current_user.public_send(column_data[:display_check])
```
That will enforce that we're asking the current user (which should the current behavior, but isn't explicit) and should fail if a `nil` is returned, which I _think_ it should.